apps/meteor/app/livechat/server/methods/saveInfo.js

```diff
@@ -6,9 +6,16 @@ import { hasPermission } from '../../../authorization';
 import { LivechatRooms } from '../../../models/server';
 import { callbacks } from '../../../../lib/callbacks';
 import { Livechat } from '../lib/Livechat';
+import { methodDeprecationLogger } from '../../../lib/server/lib/deprecationWarningLogger';
 
+/**
+ * @deprecated since version 6.0.0
```

Comment:
it is actually deprecated NOW but will be **removed** on version 6.0.0 😬

Reply:
Oops.. that's a mistake. Let me fix that :+1: